xml/Microsoft.Extensions.Logging.Console/SimpleConsoleFormatterOptions.xml

```diff
@@ -72,7 +72,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>When <see langword="true" />, the entire message gets logged in a single line.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <summary>Indicates whether the entire message gets logged in a single line or not.</summary>
        <value><see langword="true" /> to log the entire message in a single line; <see langword="false" /> otherwise..</value>
```